Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "elba-backend"
+version = "0.1.0"
+authors = ["andylokandy <andylokandy@hotmail.com>"]
+
+[dependencies]
+env_logger = "0.5"
+actix = "0.7"
+actix-web = { version = "0.7", features = ['tls']}
+bytes = "0.4.9"
+dotenv = "0.13.0"
+num_cpus = "1.0"
+futures = "0.1"
+failure_derive = "0.1"
+failure = "0.1.2"
+semver = "0.9.0"
+lazy_static = "1"
+tokio = "0.1"
+base64 = "0.9.2"
+serde = "1.0"
+serde_json = "1.0"
+serde_derive = "1.0"
+tar = "0.4"
+diesel = { version = "1.3.2", features = ["postgres", "serde_json", "r2d2"] }
+elba = { git = "https://github.com/elba/elba.git" }
```

Comment:
elba is a published crate so we can just use the published version in the registry

Reply:
Yeah, it should be, in production. I refer to the git repo just for development. For example, as the `Manifest` struct misses the description filed, I may tweak the elba repo to add it, and it will synchronize the newest changes between elba and website repo.